spec/validations_spec.cr

```diff
@@ -21,6 +35,17 @@ private class TestValidationUser
     self
   end
 
+  def run_validations_with_message
+    validate_required city, state, message: "ugh"
+    validate_inclusion_of state, in: ["CA, NY"], message: "that one's not allowed"
+    validate_confirmation_of name, with: name_confirmation, message: "name confirmation must match"    
+  end
+
+  def run_validations_with_message_callables
+    validate_required city, state, message: ->(field_name : String, field_value : String){ "#{field_name} required message from Proc" }
+    validate_inclusions_of state, in: ["CA, NY"], message: CallableMessage.new(@name)
```

Comment:
I think if you rebase this will be `validate_inclusion_of` without the "s" in inclusions